data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import os
+import psycopg2
+import getpass
+
+# for connecting to remote database
+# conn = psycopg2.connect(database='yourdb', user='dbuser', password='abcd1234', host='server', port='5432', sslmode='require')
+# for creating local database initially
+# cur.execute("CREATE TABLE Jobs(Id SERIAL PRIMARY KEY, Title TEXT, Description TEXT, Company TEXT, Location TEXT, SalaryMin INTEGER, PostURL TEXT)")
+# con.commit()
+
+# make ZipRecruiter API call
+API_KEY = os.getenv("ZIPRECRUITER_API_KEY")
+if API_KEY != None:
```

Comment:
I think it should be possible to do `if API_KEY:`.

Reply:
Also you probably want to add a print statement in the else case; otherwise this will just fail silently.